src/ensembl/src/content/app/browser/Browser.test.tsx

```diff
@@ -59,85 +59,95 @@ jest.mock('ensembl-genome-browser', () => {
 });
 jest.mock('src/gql-client', () => ({ client: jest.fn() }));
 
-const defaultProps: BrowserProps = {
-  activeGenomeId: faker.lorem.words(),
-  activeEnsObjectId: faker.lorem.words(),
-  browserActivated: false,
-  browserNavOpenState: false,
-  browserQueryParams: {},
-  chrLocation: createChrLocationValues().tupleValue,
-  isDrawerOpened: false,
-  isTrackPanelOpened: false,
-  exampleEnsObjects: [],
-  toggleTrackPanel: jest.fn(),
-  toggleDrawer: jest.fn(),
-  viewportWidth: BreakpointWidth.DESKTOP
+const mockState = createMockBrowserState();
+
+const mockStore = configureMockStore([thunk]);
+
+let store: ReturnType<typeof mockStore>;
+
+const wrapInRedux = (state: typeof mockState = mockState) => {
+  store = mockStore(state);
+  return render(
+    <Provider store={store}>
+      <Browser />
+    </Provider>
+  );
 };
```

Comment:
Names. Here and in other test files. The api of the helper function above is:

`wrapInRedux(state) -> renderResult`.

The immediate question is, what's being wrapped in redux? This name would make sense if the function were receiving a React component that it was wrapping in redux (`wrapInRedux(component) -> renderResult`), but, as proposed, the function, if you read its name, wraps in redux... the state?

The simplest fix would be to rename this function to something like `renderBrowserComponent`. A more difficult fix would be to create a true, generalised, reusable `wrapInRedux` function that would accept a component, its props, and redux state, and return the component wrapped in Redux (might be difficult with typescript).